src/__tests__/index.test.ts

```diff
@@ -296,193 +303,321 @@ describe("SDK", () => {
-      };
-
-      const gas = 100000;
-      const estimateGasMocked = vi.fn(() => {
-        return gas;
-      });
-      const transactionHash = 1234567890;
-      const sendMocked = vi.fn(() => {
-        return transactionHash;
-      });
-      const swapAndBridgeMocked = vi.fn(() => {
-        return {
-          estimateGas: estimateGasMocked,
-          send: sendMocked,
+      test("Should return txId after sending TRX to GRL", async () => {
+        const receiveFeeRequest: ReceiveTransactionCostRequest = {
+          sourceChainId: 4,
+          destinationChainId: 2,
+          messenger: Messenger.ALLBRIDGE,
+        };
+        scope
+          .post("/receive-fee", getRequestBodyMatcher(receiveFeeRequest))
+          .reply(201, receiveFeeResponse)
+          .persist();
+        /* cSpell:disable */
+        const fromAccountAddress = "TSmGVvbW7jsZ26cJwfQHJWaDgCHnGax7SN";
+        const fromTokenAddress = "TS7Aqd75LprBKkPPxVLuZ8WWEyULEQFF1U";
+        const toTokenAddress = "0xC7DBC4A896b34B7a10ddA2ef72052145A9122F43";
+        const toAccountAddress = "0x68D7ed9cf9881427F1dB299B90Fd63ef805dd10d";
+        /* cSpell:enable */
+        const sendParams: SendParamsWithChainSymbols = {
+          amount: "1.33",
+
+          fromChainSymbol: ChainSymbol.TRX,
+          fromTokenAddress: fromTokenAddress,
+          fromAccountAddress: fromAccountAddress,
+
+          toChainSymbol: ChainSymbol.GRL,
+          toTokenAddress: toTokenAddress,
+          toAccountAddress: toAccountAddress,
+
+          messenger: Messenger.ALLBRIDGE,
         };
-      });
-      const bridgeMocked = {
-        methods: {
-          swapAndBridge: swapAndBridgeMocked,
-        },
-      };
 
-      const getContract = vi.spyOn(TronBridge.prototype as any, "getContract");
-      getContract.mockImplementation(() => {
-        return bridgeMocked;
+        const gas = 100000;
+        const estimateGasMocked = vi.fn(() => {
+          return gas;
+        });
+        const transactionHash = 1234567890;
+        const sendMocked = vi.fn(() => {
+          return transactionHash;
+        });
+        const swapAndBridgeMocked = vi.fn(() => {
+          return {
+            estimateGas: estimateGasMocked,
+            send: sendMocked,
+          };
+        });
+        mockTronContract({
+          swapAndBridge: swapAndBridgeMocked,
+        });
+        const verifyTxMocked = mockTronVerifyTx();
+        // prettier-ignore
+        /* eslint-disable-next-line */
+        const transactionResponse = await sdk.send(new TronWeb("mock", "mock"), sendParams);
+        expect(swapAndBridgeMocked).toBeCalledTimes(1);
+        expect(verifyTxMocked).toBeCalledTimes(1);
+        expect(sendMocked).toBeCalledTimes(1);
+
+        expect(swapAndBridgeMocked).lastCalledWith(
+          [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 177, 3, 136, 240, 79, 131, 49,
+            181, 154, 2, 115, 44, 193, 182, 172, 13, 112, 69, 87, 75,
+          ],
```

Comment:
Those arrays kind of hard to read and not clear where did they come from, may be use addresses you from the constants above and convert them to arrays?

Reply:
I made changes addressing the comment.